supplementary_style_guide/style_guidelines/accessibility.adoc

```diff
@@ -1,21 +1,22 @@
-[[cloud-services]]
-= Cloud services guidelines
 
-Cloud services are applications that run on a platform backed by data storage infrastructure and are hosted and managed by Red{nbsp}Hat on behalf of the customer. Cloud services provide rapid iteration and can more quickly deliver updated product functionality.
-
-This means a product is no longer running exclusively in the customer’s infrastructure but is a service running in the cloud that is operated and maintained by Red{nbsp}Hat. Red{nbsp}Hat is now responsible for ensuring the service is secure, scalable, performant, and available at all times.
+[[accessibility]]
+= Accessibility
 
-Cloud services documentation requires a shift from how documentation is typically created for an on-premise product. There is more emphasis on the content delivered within the product's user interface itself, a need to deliver content rapidly without having a set release cadence, and to adapt content to the needs of cloud service users. For example, there might not be a need for upgrade and migration content that’s typical for on-premise products.
+This section highlights some guidelines for writing accessible content. For full information about writing accessible content at Red Hat, see link:https://redhat-documentation.github.io/accessibility-guide/[_Getting started with accessibility for writers_].
```

Comment:
```suggestion
For full information about writing accessible content at Red Hat, see link:https://redhat-documentation.github.io/accessibility-guide/[_Getting started with accessibility for writers_]. Here is an overview of the accessibility guidelines.
```
Trying to omit "This section..." self-referential language, as mentioned elsewhere in this doc :-)